nextflow_schema.json

```
@@ -353,6 +353,32 @@
                     "fa_icon": "fas fa-save",
                     "description": "Specify to save intermediate temporary files to --outdir",
                     "help_text": "fARGene produces a lot of additional temporary files which in most cases won't be useful.\n\nBy specifying this parameter, the directories `tmpdir/`, `hmmsearchresults/` and `spades_assemblies/` will be also saved in the output directory for closer inspection by the user, if necessary."
+                },
+                "arg_fargene_score": {
+                    "type": "number",
+                    "help_text": "The threshold score for a sequence to be classified as a (almost) complete gene. If not pre-assigned, it is assigned by the hmm_model used based on the trade-off between sensitivity and specificity. For more details see [code documentation](https://github.com/fannyhb/fargene/blob/master/fargene_analysis/fargene_analysis.py)\n\n> Modifies tool parameter(s):\n> -  fARGene: `--score `",
```

```suggestion
                    "help_text": "The threshold score for a sequence to be classified as a (almost) complete gene. If not pre-assigned, it is assigned by the hmm_model used based on the trade-off between sensitivity and specificity. For more details see [code documentation](https://github.com/fannyhb/fargene/blob/master/fargene_analysis/fargene_analysis.py)\n\n> Modifies tool parameter(s):\n> -  fARGene: `--score`",
```
Note: We could check which range is the score (e.g. 0–1 or 0–100) and document this here.

---

I think the schema builder may allow specification of this :+1: